### PR TITLE
404 links fix

### PR DESCRIPTION
**Pull request by iArchitSharma**
This PR fixes some 404 links in issue #5238 
image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/blog/2019/2019-10-12-hacktoberfest-2019-is-heating-up/index.mdx [hunk 1]
```diff
@@ -11,6 +11,9 @@ tags:
 published: true
 redirect_from:
  - /blog/hacktoberfest-2019-heating-up/
+ - /blog/landscape/hacktoberfest-2019-heating-up
+ - /blog/landscape/hacktoberfest-2019-heating-up/
+ - /blog/landscape/hacktoberfest-2019-heating-up.html
```
Comment (leecalcote) [inline src/collections/blog/2019/2019-10-12-hacktoberfest-2019-is-heating-up/index.mdx:16]: Is the .html showing up in the 404 report?

File: gatsby-node.js [hunk 2]
```diff
@@ -72,6 +72,21 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
...
+  createRedirect({ fromPath: "/products/nighthawk", toPath: "/projects/nighthawk", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/blog/tag/lfx", toPath: "/blog/tag/programs", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/blog/tag/lfx", toPath: "/blog/tag/programs", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/blog/tag/open-service-mesh", toPath: "/blog/tag/service-mesh", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/blog/tag/open-service-mesh/", toPath: "/blog/tag/service-mesh", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/blog/tag/news", toPath: "/company/news", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/products/service-mesh-interface-conformance", toPath: "/service-mesh-landscape", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/products.html", toPath: "/products", redirectInBrowser: true, isPermanent: true });
```
Comment (leecalcote) [inline gatsby-node.js:89]: Is the .html showing up in the 404 report?

File: src/collections/blog/2019/2019-10-12-hacktoberfest-2019-is-heating-up/index.mdx (hunk 1 above)
Comment (iArchitSharma) [reply src/collections/blog/2019/2019-10-12-hacktoberfest-2019-is-heating-up/index.mdx:16]: @leecalcote yes it is

File: gatsby-node.js (hunk 2 above)
Comment (iArchitSharma) [reply gatsby-node.js:89]: yes its showing up

**Comment by vishalvivekm** [comment]
build and preview is failing here @iArchitSharma. Please fix and let's discuss this in websites call today.

File: src/collections/blog/2019/2019-10-12-hacktoberfest-2019-is-heating-up/index.mdx (hunk 1 above)
Comment (leecalcote) [reply src/collections/blog/2019/2019-10-12-hacktoberfest-2019-is-heating-up/index.mdx:16]: @iArchitSharma this PR is pending a more informed response. Where are the .html 404s coming from, specifically? A bot is making them up? Coming from the sitemap?

**Pull request closed without merging**